projects/driver_controls/inc/steering_wheel.h

```diff
@@ -0,0 +1,42 @@
+#pragma once
+
+// Module for steering wheel angle percentage reading
+//
```

Comment:
You need to mention what modules are required to be initialized before this module will work.